lib/board.rb

```diff
@@ -0,0 +1,14 @@
+class Board
+
+  attr_reader :grid
+
+  def initialize
+    @grid = [["*", "1", "2", "3"],
```

Comment:
SRP: It feels like a violation of SRP to put the numbers the user should use for inputting moves into the data representation.